src/examples/CheckBoxExamplePage.tsx

```diff
@@ -25,38 +25,57 @@ export const CheckBoxExamplePage: React.FunctionComponent<{}> = () => {
         },
       ]}>
       <Example title="A simple Checkbox." code={'<CheckBox />'}>
-        <CheckBox />
+        <CheckBox accessibilityLabel="Simple Checkbox" />
```

Comment:
https://learn.microsoft.com/en-us/windows/win32/winauto/uianameshouldnotcontaincontroltype you'll likely get hit with some of these errors. Try changing it to something like "simple example" "checked example" etc. 

Same for all of the other components.